pretext/Strings/StringMethods.ptx

```diff
@@ -54,50 +57,6 @@ print(tt)
                             Returns a string in all lowercase
                         </cell>
       </row>
-      <row>
-        <cell>
-                            capitalize
-                        </cell>
-        <cell>
-                            none
-                        </cell>
-        <cell>
-                            Returns a string with first character capitalized, the rest lower
-                        </cell>
-      </row>
-      <row>
-        <cell>
-                            strip
```

Comment:
Do we want to keep strip() in? I remember students did actually use this ones. 

I agree with deleting the rest though.

Reply:
Yeap but its needed in week 9 (file io), and we taught it again. Because of this chapter, they wrote this to their cheatsheet in exam 1 and there were lots of incorrect use of it.